packages/p2p/src/components/order-details/order-details-wrapper.jsx

```diff
@@ -8,37 +8,38 @@ import OrderDetailsFooter from 'Components/order-details/order-details-footer.js
 
 const OrderDetailsWrapper = ({ children, onPageReturn, page_title }) => {
     const { order_store, sendbird_store } = useStores();
-
     return isMobile() ? (
-        <MobileFullPageModal
-            className='order-details'
-            body_className='order-details--body'
-            height_offset='80px'
-            is_flex
-            is_modal_open
-            pageHeaderReturnFn={onPageReturn}
-            page_header_text={page_title}
-            renderPageHeaderTrailingIcon={() => (
-                <Icon
-                    icon='IcChat'
-                    height={15}
-                    width={16}
-                    onClick={() => sendbird_store.setShouldShowChatModal(true)}
-                />
-            )}
-            renderPageFooterChildren={() =>
-                order_store.order_information.should_show_order_footer && (
-                    <OrderDetailsFooter order_information={order_store.order_information} />
-                )
-            }
-        >
-            {children}
-        </MobileFullPageModal>
+        <div data-testid='order-details-wrapper-mobile'>
```

Comment:
Do we need this extra div for the mobile component just to add the test id? Can we not add it directly to the `MobileFullPageModal`?